basxconnect/mailer_integration/views.py

```diff
@@ -192,3 +192,7 @@ def post(self, request, *args, **kwargs):
             MailerPerson.from_mailing_preferences(self.object)
         )
         return result
+
+    def get_layout(self):
```

Comment:
Somehow the `EditSubscriptionView` only works if we remove this method. It was removed from main, but it seems merging this PR will add it back..?

Reply:
The correct way to render this would be:

```python
return layout.forms.Form(
    hg.C("form"), *[layout.forms.FormField(field) for field in self.fields]
)
```

But this does nothing else than the super method, as all bread views consider the ```as_ajax``` parameter already. So yes, removing it makes the most sense.